features/FxA-97-password-strength/README.md

```diff
@@ -0,0 +1,67 @@
+
+## Outcomes
+
+[comment]: # (What are the outcomes you want to achieve? What is the success criteria?)
+
+We feel that we could better serve user security and our [Data Privacy Principles](https://www.mozilla.org/en-US/privacy/principles/) by being more transparent about the risks and benefits of signing up for an account. This will hopefully result in stronger passwords as measured by our bloom filter, and also result in fewer account sign-ups (as many users will realize it's not something they need or want). Another goal is increasing multi-device usage, which in turn should result in an increase in overall Firefox engagement.
+
+## Hypothesis
+
+[comment]: # (A high level hypothesis of how the feature you're proposing is going help us achieve the outcomes listed above. I recommend this be a sentence of the form. We believe that doing this building this creating this experience for these people will achieve this outcome. We will know this is true when we see this qualitative feedback quantitative feedback KPI change.)
+
+We believe that providing more information about password strength for registering users will achieve stronger passwords. We will know this is true when we see a reduction in our bloom filter numbers.
+
+We believe that providing more information about password strength for registering users will achieve greater awareness about password strength. We will know this is true when we survey new users and ask about passwords.
+
+We believe that blocking known weak passwords will achieve stronger passwords. We will know this is true when we see a reduction in our bloom filter numbers.
+
+We believe that these changes will result in us collecting only the data that benefits users. We will know this is true when we see a slight reduction in the number of new accounts.
+
+## Metrics
+
+[comment]: # (How are you going to measure the outcome / success? Please provide sample artifact graphs here.)
+
+* Datadog Dashboard › FxA Content Server - Password Strength › % of users that hit each restriction
+ * Fewer users hitting ALL_LETTERS_OR_NUMBERS and BLOOMFILTER_HIT
+* Datadog Dashboard › FxA Content Server - Registration › % of too short passwords
+ * Fewer users hitting TOO_SHORT
+* Datadog Dashboard › FxA Content Server - Registration › Signup funnel
+ * Slightly reduced signup rate
+
+## Detailed design
+
+[comment]: # (This is the bulk of the RFC. Explain the design in enough detail for somebody familiar with the language to understand. This should get into specifics and corner-cases, and include examples of how the feature is used.)
+
+![Password Strength](password-strength.png)
+
+#### Password field give focus
+When the user gives focus to the password field on the registration form, show instructions below that say:
+* A strong, unique password will keep your Firefox data safe from intruders. <u>More info</u>.
```

Comment:
Of course, this text here could [negate the increase I think could happen](https://github.com/mozilla/fxa/pull/150/files#r64023087).